.github/pull_request_template.md

```diff
@@ -0,0 +1 @@
+- [ ] spec.md has been updated or doesn't need to updated
```

Comment:
Is there anything we can add that will process the spec programmatically? I feel we've done this elsewhere. Alternatively  we should add support in porcelain for doing openapi/swagger as that will allow examples to be included and tested more easily